src/overlays/actors/ovl_En_Fall/z_en_fall.h

```diff
@@ -3,14 +3,54 @@
 
 #include "global.h"
 
+#define EN_FALL_SCALE(thisx) ((thisx)->params & 0x7F)
+#define EN_FALL_TYPE(thisx) (((thisx)->params & 0xF80) >> 7)
+
+#define EN_FALL_FLAG_FIRE_BALL_INTENSIFIES (1 << 0)
+#define EN_FALL_FLAG_FIRE_RING_APPEARS (1 << 1)
+
+typedef enum {
+    EN_FALL_TYPE_TERMINA_FIELD_MOON,
+    EN_FALL_TYPE_TITLE_SCREEN_MOON,
+    EN_FALL_TYPE_CRASHING_MOON,
+    EN_FALL_TYPE_CRASH_FIRE_BALL,
+    EN_FALL_TYPE_CRASH_RISING_DEBRIS,
+    EN_FALL_TYPE_LODMOON_NO_LERP, // unused in final game
+    EN_FALL_TYPE_LODMOON,
+    EN_FALL_TYPE_STOPPED_MOON_CLOSED_MOUTH,
+    EN_FALL_TYPE_MOONS_TEAR,
+    EN_FALL_TYPE_CLOCK_TOWER_MOON,
+    EN_FALL_TYPE_STOPPED_MOON_OPEN_MOUTH,
+    EN_FALL_TYPE_CRASH_FIRE_RING,
+    EN_FALL_TYPE_LODMOON_INVERTED_STONE_TOWER,
+} EnFallType;
```

Comment:
```suggestion
typedef enum {
    /* 0 */ EN_FALL_TYPE_TERMINA_FIELD_MOON,
    /* 1 */ EN_FALL_TYPE_TITLE_SCREEN_MOON,
    EN_FALL_TYPE_CRASHING_MOON,
    EN_FALL_TYPE_CRASH_FIRE_BALL,
    EN_FALL_TYPE_CRASH_RISING_DEBRIS,
    EN_FALL_TYPE_LODMOON_NO_LERP, // unused in final game
    EN_FALL_TYPE_LODMOON,
    EN_FALL_TYPE_STOPPED_MOON_CLOSED_MOUTH,
    EN_FALL_TYPE_MOONS_TEAR,
    EN_FALL_TYPE_CLOCK_TOWER_MOON,
    EN_FALL_TYPE_STOPPED_MOON_OPEN_MOUTH,
    EN_FALL_TYPE_CRASH_FIRE_RING,
    EN_FALL_TYPE_LODMOON_INVERTED_STONE_TOWER,
} EnFallType;
```
Could you fill in the offsets for this enum?

Reply:
Whoops, forgot to do this, thanks for pointing it out.